js/controllers/grid/users/stageParticipant/StageParticipantGridHandler.js

```diff
@@ -40,6 +41,7 @@
 					.each(function() {
 						$.pkp.classes.Handler.getHandler($(this)).reload();
 					});
+			pkp.eventBus.$emit('decisionActionUpdated');
```

Comment:
Just to make sure: here you're updating the decision div whenever a participant is added or removed. Is that right?

Reply:
Yes, it is probably a very rare case, thus maybe also not needed: if one assignes himself/herself as a recommendOnly editor, he/she should see the "Make Recommendatin" button and the same if he/she removes himself/herself... -- I can also leave it so that such a user has to reload the page?

Reply:
No that sounds good. Best to just reload it to keep the data fresh.

Reply:
Hmmm... the #reviewDecisionsDiv-13 is reloaded in the line above but what that is... Hmmm... I think I will have to test this once again... Sorry...

Reply:
And does this mean that this reload in the line above is bound to all 'dataChanged' events?

Reply:
OK, this is not necessary here... I will change it...